spec/models/ext_management_system_spec.rb

```diff
@@ -258,6 +258,24 @@
     before(:each) do
       @ems = FactoryGirl.create(:ems_vmware)
       (1..2).each { |i| FactoryGirl.create(:vm_vmware, :ext_management_system => @ems, :name => "vm_#{i}") }
+      FactoryGirl.create(
+        :host_vmware,
+        :ext_management_system => @ems, :name => "host_1",
+        :hardware => FactoryGirl.create(:hardware, :cpu_total_cores => 4, :memory_mb => 1024)
+      )
+      FactoryGirl.create(
+        :host_vmware,
+        :ext_management_system => @ems, :name => "host_2",
+        :hardware => FactoryGirl.create(:hardware, :cpu_total_cores => 2, :memory_mb => 2048)
+      )
+    end
+
+    it "#total_cores" do
+      expect(@ems.total_cores).to eq(6)
+    end
+
+    it "#total_memory" do
+      expect(@ems.total_memory).to eq("3 GB")
```

Comment:
again, would like this to return `3` or `3 * 1024 * 1024` - the screen formatting doesn't belong in the model